jx_module/variables.tf

```diff
@@ -0,0 +1,174 @@
+variable "cluster_name" {
+  type    = string
+  default = ""
```

Comment:
Would you please add a default valid name instead of an empty string?

Reply:
The default values are generated by the random provider inside local.tf.  These need to be empty strings here to stop terraform forcing input of them (or a vars file with empty values). There are no sensible defaults here and we're better placed randomising these values where they are not supplied by the user.